package.json

```
@@ -45,9 +45,9 @@
     "typescript": "^3.4.2"
   },
   "dependencies": {
-    "@holochain/hcid-js": "0.0.5",
+    "@holochain/hcid-js": "0.0.6",
     "dialog-polyfill": "^0.4.10",
-    "jquery": "^3.3.1",
+    "jquery": ">=3.4.0",
```

Might want to stick with the tidle, won't that break things when jQuery 4 comes out?
```suggestion
    "jquery": "^3.4.0",
```